keps/sig-storage/20191125-bucket-provisioning.md

```diff
@@ -554,6 +555,8 @@ Upon success, the `BucketAccess` instance is ready and the app workload can acce
 + User creates a `BucketRequest` (BR) that references this `Bucket`.
 + User creates `BucketAccessRequest` (BAR) that references their BR, a `BucketAccessClass`, and their secret containing access tokens.
 + COSI detects the existence of the BAR and follows the [grant access](#grant-bucket-access) steps.
++ COSI detects the existence of the BAR, BA and executes a validation process.
```

Comment:
So when COSI detects a new BAR does it follow _both_ the validation process and the "grant access" steps from the bullet above? It reads that way to me.

Reply:
Do you see an issue here?

Reply:
No issue. Mostly want to make sure I understand it.
I wasn't sure if validation was only for static use-cases.